addons/pharma/functions/fnc_treatmentAdvanced_PervitinLocal.sqf

```diff
@@ -19,380 +20,332 @@ params ["_patient"];
-},
-[_patient],
-510] call CBA_fnc_waitAndExecute; /// 8:30m
-
-
-
-[{
-	params ["_patient"];
-
-	["PDF"] call ace_advanced_fatigue_fnc_removeDutyFactor;
-	[LLSTRING(Pervitin_end), 2, _patient] call ACEFUNC(common,displayTextStructured);
-},
-[_patient],
-600] call CBA_fnc_waitAndExecute; /// 10m
-
-
+	[{
+		params ["_patient"];
 
+		ace_advanced_fatigue_anReserve = ace_advanced_fatigue_anReserve + 3000;
+		["PDF", 0] call ace_advanced_fatigue_fnc_addDutyFactor;
+		[LLSTRING(Pervitin_start), 2, _patient] call ACEFUNC(common,displayTextStructured); 
+	},
+	[_patient],
+	10] call CBA_fnc_waitAndExecute;
 
 
+	[{
+		params ["_patient"];
 
- /// ACE Fatigue Weapon Sway
+		["PDF"] call ace_advanced_fatigue_fnc_removeDutyFactor;
+		[LLSTRING(Pervitin_mid), 2, _patient] call ACEFUNC(common,displayTextStructured);
+	},
+	[_patient],
+	120] call CBA_fnc_waitAndExecute; /// 2m
 
-if (isNil "TAG_originalSwayFactor")    then {TAG_originalSwayFactor = ace_advanced_fatigue_swayFactor;};
 
 	[{
-	params ["_patient"];
-	ace_advanced_fatigue_swayFactor = TAG_originalSwayFactor * 0.3;
-},
-[_patient],
-15] call CBA_fnc_waitAndExecute;
+		params ["_patient"];
 
+		ace_advanced_fatigue_anReserve = ace_advanced_fatigue_anReserve + 3000;
+		["PDF", 0.4] call ace_advanced_fatigue_fnc_addDutyFactor;
+		[LLSTRING(Pervitin_mid2), 2, _patient] call ACEFUNC(common,displayTextStructured);
+	},
+	[_patient],
+	180] call CBA_fnc_waitAndExecute; /// 3m
 
 
 	[{
-	params ["_patient"];
-	ace_advanced_fatigue_swayFactor = TAG_originalSwayFactor * 0.5 ;
-},
-[_patient],
-60] call CBA_fnc_waitAndExecute;
+		params ["_patient"];
 
-	
+		ace_advanced_fatigue_anReserve = ace_advanced_fatigue_anReserve + 3000;
+		["PDF", 0.6] call ace_advanced_fatigue_fnc_addDutyFactor;
+		[LLSTRING(Pervitin_mid3), 2, _patient] call ACEFUNC(common,displayTextStructured);
+	},
+	[_patient],
+	360] call CBA_fnc_waitAndExecute; /// 6m
 
-[{
-	params ["_patient"];
-	ace_advanced_fatigue_swayFactor = TAG_originalSwayFactor * 1;
-},
-[_patient],
-90] call CBA_fnc_waitAndExecute;
 
+	[{
+		params ["_patient"];
 
+		["PDF", 2] call ace_advanced_fatigue_fnc_addDutyFactor;
+		[LLSTRING(Pervitin_mid4), 2, _patient] call ACEFUNC(common,displayTextStructured);
+	},
+	[_patient],
+	510] call CBA_fnc_waitAndExecute; /// 8:30m
 
-[{
-	params ["_patient"];
-	ace_advanced_fatigue_swayFactor = (TAG_originalSwayFactor * 1.3) min 2;
-},
-[_patient],
-120] call CBA_fnc_waitAndExecute; /// 2m
 
+	[{
+		params ["_patient"];
 
+		["PDF"] call ace_advanced_fatigue_fnc_removeDutyFactor;
+		[LLSTRING(Pervitin_end), 2, _patient] call ACEFUNC(common,displayTextStructured);
+	},
+	[_patient],
+	600] call CBA_fnc_waitAndExecute; /// 10m
 
-[{
-	params ["_patient"];
-	ace_advanced_fatigue_swayFactor = (TAG_originalSwayFactor * 1.5) min 2;
-},
-[_patient],
-150] call CBA_fnc_waitAndExecute;
 
+	/// ACE Fatigue Weapon Sway
 
+	if (isNil "KAM_originalSwayFactor")    then {
```

Comment:
small note because i didnt check code that much yesterday -> instead of "KAM_originalSwayFactor" use QGVAR(originalSwayFactor)